src/test/java/org/cloudfoundry/promregator/SpringBootLoadPropertiesForTestingSpringApplication.java

```diff
@@ -21,4 +24,8 @@ public Object anything() {
 	public void check() {
 		Assert.assertTrue(dummyValue);
 	}
+
+	public String getEncryptedValue() {
```

Comment:
Sorry for sounding nit-picky: `getEncryptedValue` sounds to me as if the value I get back is still encrypted 😀 (well, my brain might be biased having studied cryptography some years back...). Naming it `getPlaintextValue()` might be more correct (it's what cryptography would say), but could also be confusing - especially, if we keep the attributes name as `encryptedValue`, which would awkwardly look strange:

```java
public String getPlaintextValue() {
    return encryptedValue;
}
```
😨 

In fact the whole story about this `'{cipher}...'` thing is that it should be transparent to the `SpringConfiguration`.

What do you think about changing the spring property's name from `encrypted.value` to `secret.value` (not describing how it was done, but for what it - typically - is used) and then calling this method `getSecretValue()` here? IMO this would avoid this weird discussion altogether.

Reply:
It's not nitpicking. I appreciate the clear idea of how the code should be written. I changed it to `getSecretValue`.